<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>821</width>
      <height>501</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionSelect_None"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionAnimals"/>
    <addaction name="actionShapes"/>
    <addaction name="actionFood"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolbar_top"/>
    <addaction name="actionToolbar_bottom"/>
    <addaction name="actionToolbar_left"/>
    <addaction name="actionToolbar_right"/>
    <addaction name="separator"/>
    <addaction name="actionToolbar_floatable"/>
    <addaction name="actionToolbar_movable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInsert"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/new.png</normaloff>:/files/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/open.png</normaloff>:/files/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/save.png</normaloff>:/files/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/saveas.png</normaloff>:/files/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/copy.png</normaloff>:/files/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/cut.png</normaloff>:/files/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/paste.png</normaloff>:/files/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/selectall.png</normaloff>:/files/images/selectall.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionSelect_None">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/selectnone.png</normaloff>:/files/images/selectnone.png</iconset>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
  </action>
  <action name="actionAnimals">
   <property name="text">
    <string>Animals</string>
   </property>
  </action>
  <action name="actionShapes">
   <property name="text">
    <string>Shapes</string>
   </property>
  </action>
  <action name="actionFood">
   <property name="text">
    <string>Food</string>
   </property>
  </action>
  <action name="actionToolbar_top">
   <property name="text">
    <string>Toolbar top</string>
   </property>
  </action>
  <action name="actionToolbar_bottom">
   <property name="text">
    <string>Toolbar bottom</string>
   </property>
  </action>
  <action name="actionToolbar_left">
   <property name="text">
    <string>Toolbar left</string>
   </property>
  </action>
  <action name="actionToolbar_right">
   <property name="text">
    <string>Toolbar right</string>
   </property>
  </action>
  <action name="actionToolbar_floatable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar floatable</string>
   </property>
  </action>
  <action name="actionToolbar_movable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar movable</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
